qemu/tests/virtio_fs_hotplug.py

```diff
@@ -141,11 +141,15 @@ def start_vfs_service(session):
                     if s:
                         test.fail("Fail command: %s. Output: %s" % (reg_cmd, o))
                 error_context.context("Reboot guest.", test.log.info)
-                session = vm.reboot()
+                vm.reboot()
+                # After reboot, 'session' is changed.
+                # The 'session' needs to be obtained from vm
+                session = vm.wait_for_login()
```

Comment:
we don't need to log into vm again for vm.reboot will return a new session object, we can use that.